app/components/bulk_actions_form_component.rb

```diff
@@ -29,14 +29,14 @@ def action_types
         ["Apply APO defaults", new_apply_apo_defaults_job_path(search_of_druids)],
         ["Set object rights", new_rights_job_path(search_of_druids)],
         ["Edit license and rights statements", new_license_and_rights_statement_job_path(search_of_druids)],
-        ["Edit catkeys and barcodes", new_catkey_and_barcode_job_path(search_of_druids)],
-        ["Refresh MODS from catkey", new_refresh_mods_job_path(search_of_druids)],
+        ["Edit catalog record IDs and barcodes", new_catalog_record_id_and_barcode_job_path(search_of_druids)],
```

Comment:
This is the first instance of attempting to move away from using `catkey` everywhere all the time, so a big chunk of the changes in this branch are renames to be less coupled to implementation details.

Reply:
uh - this is up to the POs

Reply:
uh?

Reply:
I'm reading this as:

> Consider using `CatalogRecordId.label` (catkey / folio id) instead of `catalog record ID` in user-facing changes, or ask @andrewjbtw and @arcadiafalcone how they feel about this change.